### ISSUE TITLE
Several vulnerabilities for accel-ppp

### ISSUE THREAD

**Issue by hac425xxx** [xebd/accel-ppp#131]
I try to send email to contact@accel-ppp.org and  dima@accel-ppp.org, but my email is blocked.

The vulnerability report is attached to the email. 
Sorry, it is in Chinese. 
You can try to use Google Translate. 
If you have any questions, please contact me

Wait for your reply

[vuln.zip](https://github.com/xebd/accel-ppp/files/4461479/vuln.zip)

**Comment by DmitriyEshenko** [comment]
Hi @hac425xxx 
Thanks, very informative. The patch will ready soon.
Can you remove the attached file?

**Comment by hac425xxx** [comment]
removed, and could you please request some cve for these bug, you can requset in github project

```
https://github.com/xebd/accel-ppp/security
```
the document for requtest
```
https://help.github.com/en/github/managing-security-vulnerabilities/publishing-a-security-advisory

```

**Comment by DmitriyEshenko** [comment]
Thanks @hac425xxx 
Patches already available in org repo https://github.com/accel-ppp/accel-ppp/commits/master
@xebd can you merge and create CVE request?

**Comment by DmitriyEshenko** [comment]
Hello @hac425xxx , can you check this version
https://github.com/xebd/accel-ppp/commit/ed7b28722ec8513838b49699a862a69055c8c596

**Comment by Neustradamus** [comment]
@DmitriyEshenko: The CVE fix has been added in this original repository?

**Comment by DmitriyEshenko** [comment]
Yes, you can easily check by commit id https://github.com/accel-ppp/accel-ppp/commit/ed7b287
